src/Filters/Url.js

```diff
@@ -1,14 +1,15 @@
 const validUrl = require("valid-url");
 const TemplatePath = require("../TemplatePath");
 
-module.exports = function(url, pathPrefix) {
+module.exports = function (url, pathPrefix) {
   // work with undefined
   url = url || "";
 
   if (
     validUrl.isUri(url) ||
     url.indexOf("http://") === 0 ||
-    url.indexOf("https://") === 0
+    url.indexOf("https://") === 0 ||
+    (url.indexOf("//") === 0 && url !== "//")
```

Comment:
This could also be written as `(url.startsWith("//") && url !== "//")` 
Actually all of them in the conditional can be replaced with `startsWith()` as there is no way the url to contain the protocol in any other position except the... start of the string